docs/editing-components.md

```diff
@@ -0,0 +1,380 @@
+   - `help_text='Some text'` if the field needs a more verbose explanation to
+     be shown in the editor to make it clear to users how it should work
+   - `default=<some appropriate value>` if you want the field to have a
+     specific default value, e.g., `True` to have a `BooleanBlock` checkbox
+     default to checked.
+   - Certain blocks may take other arguments, as described in the
+     [basic Wagtail blocks documentation](https://docs.wagtail.io/en/v1.13.4/topics/streamfield.html#basic-block-types).
+1. [Edit the component template](#the-html-template) to do something with the
+   field's data – output it, use it to trigger a CSS class, etc.
+1. [Create a schema migration.](#creating-migrations-for-streamfield-blocks)
+
+
+### Editing a field
+
+1. [Determine if the change you want to make will need a data migration.](#you-may-also-need-a-data-migration)
+   - If the answer is **no**: make your changes,
+     [create a schema migration](#creating-migrations-for-streamfield-blocks), and be on your merry way.
+   - If the answer is **yes**: continue on.
+1. [Add the new version of the field.](#adding-a-field)
+1. [Create a schema migration](#creating-migrations-for-streamfield-blocks) for adding the new field.
+1. [Create a data migration](#you-may-also-need-a-data-migration)
+   to copy data from the old field into the new field.
+1. [Edit the component template](#the-html-template)
+   to use the new field's data instead of the old field's data.
+1. [Remove the old field.](#removing-a-field)
+1. [Create a schema migration](#creating-migrations-for-streamfield-blocks) for removing the old field.
+
+
+### Removing a field
+
+These instructions presume that you do not care
+about any data stored in the field you are deleting.
+If that is not the case, please go up to the
+[instructions for editing a field](#editing-a-field)
+and come back here when instructed.
+
+1. Locate the field you want to remove in the block's Python class.
+1. Delete the field definition.
+1. [Create a schema migration.](#creating-migrations-for-streamfield-blocks)
+
+
+### Creating migrations for StreamField blocks
+
+To automatically generate a **schema migration**, run
+`./cfgov/manage.py makemigrations -n <description_of_changes>`
+from the root of the repository.
+
+#### You may also need a data migration
+
+Some field edits (like changing the `default`, `label`, `help_text`,
+and `required` properties, or changing the order of fields on a block)
+do not require a data migration. A schema migration is sufficient.
+
+However, if you…
+
+- … are renaming an existing field …
+- … are deleting a field (or converting it to a new type of field)
+  and don't want to lose any existing data stored in that field …
+- … are renaming a block within a page's StreamField definition …
+- … are deleting a block (or converting it to a new type of block)
+  and don't want to lose any existing data stored in that field …
+
+… then you will need a data migration!
```

Comment:
Same as with this list in the migration doc (below — reviewed this files in reverse order).

Reply:
Addressed in 968e783c4997ad98d51c1f8310159a1a68772b07